pkg/app/api/service/webservice/service.proto

```diff
@@ -401,3 +405,15 @@ message ListAPIKeysRequest {
 message ListAPIKeysResponse {
     repeated model.APIKey keys = 1;
 }
+
+message GetInsightDataRequest {
+    string application_id = 1 [(validate.rules).string.min_len = 1];
+    pipe.model.InsightMetricsKind metrics_kind = 2;
+    pipe.model.InsightStep insight_step = 3;
```

Comment:
nit: I think "step" is fine. We are also using "metrics_kind" instead of "insight_metrics_kind"